### PR TITLE
Update LlamaEntity.mapping

### PR DESCRIPTION
**Pull request by tyra314**
Adds mappings for two more methods

### CODE REVIEW

File: mappings/net/minecraft/entity/passive/LlamaEntity.mapping [hunk 1]
```diff
@@ -19,3 +19,7 @@ CLASS arv net/minecraft/entity/passive/LlamaEntity
 		ARG 0 color
 	METHOD u setVariant (I)V
 	METHOD v setStrength (I)V
+	METHOD h spitAt (Laio;)V
+		ARG 1 target
+	METHOD a breedWith (Laid;)Larv;
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/LlamaEntity.mapping:24]: Looks like `createChild` to me by param and return types.

Comment (tyra314) [reply mappings/net/minecraft/entity/passive/LlamaEntity.mapping:24]: There is already a different function literally called `createChild`, which is used in this function. Also, there is a function `canBreedWith`.

Comment (liach) [reply mappings/net/minecraft/entity/passive/LlamaEntity.mapping:24]: what does this do in addition to calling create child?

Comment (tyra314) [reply mappings/net/minecraft/entity/passive/LlamaEntity.mapping:24]: createChild gives a plain new Entity instance. This function, however, does the inherit-stats-from-parents thing and gives the child the color of one of the parents. One parent is the instance, the function is called on and the other is the parameter.

**Comment by modmuss50** [comment]
merging, any issues should go to a new PR or an issue.

**Pull request merged**